src/components/LandingPage/LandingPage.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { LandingPage } from './LandingPage';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof LandingPage> = {
+  title: 'Components/LandingPage',
+  component: LandingPage,
+  parameters: {
+    layout: 'fullscreen',
+    docs: {
+      description: {
+        component:
```

Comment:
Story description is written in Spanish/hardcoded text. Project requires component strings to be localized and code/comments in English.

**Sugerencia:**
```typescript
Convert story descriptions and any code comments to English and avoid hardcoding localized strings in stories; use i18n resources if story content needs localization.
```